tests/bootstrap.php

```diff
@@ -9,6 +9,8 @@
 error_reporting(E_ALL | E_STRICT);
 ini_set('max_execution_time', 900);
 ini_set('date.timezone', 'GMT+0');
+//ini_set('apc.enable_cli', false);
+ini_set('opcache.jit', 0); // don't JIT
```

Comment:
what does it break when it's enabled?

Reply:
@falkenhawk It breaks pretty much everything, but JIT is not enabled by default - I had it enabled for testing.